applications/MeshingApplication/CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ endif(${KRATOS_BUILD_TESTING} MATCHES ON)
 ###############################################################
 ## define library Kratos which defines the basic python interface
 add_library(KratosMeshingApplication SHARED ${KRATOS_MESHING_APPLICATION_SOURCES} ${KRATOS_MESHING_APPLICATION_TESTING_SOURCES})
-target_link_libraries(KratosMeshingApplication KratosCore triangle tetMeshOpt)
+target_link_libraries(KratosMeshingApplication KratosCore KratosStructuralMechanicsApplication triangle tetMeshOpt)
```

Comment:
is this a good idea?
Does one then have to always compile StructuralMechanics to compile the MeshingApplication?

Reply:
Certainly I don't know, that is a question I have to @roigcarlo, in any case the other alternative would be to remove the utility that uses the process in the meshing application (it is deprecated)

Reply:
Done, now it compiles the utility just when the StructuralMechanics is compiled